README.md

```diff
@@ -1,72 +1,48 @@
 
 ### Pulumi
 
-Pulumi is an open source infrastructure as code tool for creating, deploying, and managing cloud infrastructure. Pulumi works with traditional infrastructure like VMs, networks, and databases, in addition to modern architectures, including containers, Kubernetes clusters, and serverless functions.
-
-Pulumi uses real languages for infrastructure as code, which means many benefits: IDEs, abstractions including functions, classes, and packages, existing debugging and testing tools, and more. The result is greater productivity with far less copy and paste, and it works the same way no matter which cloud you're targeting.
-
-For more information, please refer to this [link](https://www.pulumi.com/docs/intro/concepts/)
-
-#### Stack
-
-Every Pulumi program is deployed to a stack. A stack is an isolated, independently configurable instance of a Pulumi program. Stacks are commonly used to denote different phases of development (such as development, staging and production) or feature branches (such as feature-x-dev, jane-feature-x-dev). For more information, please refer to this [link](https://www.pulumi.com/docs/intro/concepts/stack/).
-
-In our project, we are considering that each git branch is a stack. For more information, please refer to this [link](https://www.pulumi.com/docs/intro/concepts/organizing-stacks-projects/).
-
-#### Configuration and Secrets
-
-As we are considering that each branch is a stack, we can handle the environment variables as such following this [documentation](https://www.pulumi.com/docs/intro/concepts/stack/). However, in my opinion, secrets such as API Keys are not yet handle in a good way by Pulumi. Therefore, we are sending API Keys using `.env` file in development environment and GitHub Secrets (or similar feature) in our CI/CD environment. Check the section `How to Use` for more information.
-
-#### State and Backends
-
-Pulumi stores its own copy of the current state of your infrastructure. This is often simply called state, and is stored in transactional snapshots we call checkpoints. A checkpoint is recorded by Pulumi at various points so that it can operate reliably — whether that means diffing goal state versus current state during an update, recovering from failure, or destroying resources accurately to clean up afterwards. Because state is critical to how Pulumi operates, we’ll cover a few of the state backend options on this page.
-
-Pulumi supports multiple backends for storing your infrastructure state:
-
-- The Pulumi Service backend
-- A self-managed backend, either stored locally on your filesystem or remotely using a cloud storage service
-
-For more information, check this [link](https://www.pulumi.com/docs/intro/concepts/state/).
```

Comment:
@bermudezmt I like the way you summarised about Pulumi. Did you remove all this block because you think that can be read directly from Pulumi docs? How about we add something like: 

=====

Key Pulumi concepts you should be familiat with before starting using this provider are: 
* [Stack](https://www.pulumi.com/docs/intro/concepts/stack/)
* [Configurations](https://www.pulumi.com/docs/intro/concepts/config/) and [secrets](https://www.pulumi.com/docs/intro/concepts/secrets/)
* [State](https://www.pulumi.com/docs/intro/concepts/state/)

===

Reply:
There you go, via 7ab0f3f78f091e5d96e9c4926daac62f961c2293. Thank you so much, @vernig .